src/main/io/osd.c

```diff
@@ -410,8 +461,14 @@ static void osdDrawSingleElement(uint8_t item)
 
     case OSD_ALTITUDE:
         {
-            const int32_t alt = osdGetMetersToSelectedUnit(getEstimatedAltitude());
-            tfp_sprintf(buff, "%c%d.%01d%c", alt < 0 ? '-' : ' ', abs(alt / 100), abs((alt % 100) / 10), osdGetMetersToSelectedUnitSymbol());
+            int32_t alt = osdGetMetersToSelectedUnit(getEstimatedAltitude());
```

Comment:
I kind of prefer the implementation I proposed in #3615 here, that keeps the negative sign aligned next to the number.